app/views/green_lanes/eligibilities/new.html.erb

```diff
@@ -40,7 +40,7 @@
         legend: { text: "Are you authorised to trade under the UK Internal Market Scheme (UKIMS)?", size: "m" },
         hint: { text: "SPIMM is only available to UKIMS authorised traders." } %>
 
-      <%= f.govuk_submit "Continue", class: "govuk-button" %>
+      <%= f.govuk_submit "Continue", class: "govuk-button govuk-!-margin-top-6 govuk-!-margin-bottom-4" %>
```

Comment:
Spacing changes here changed because the other buttons around this object provide extra margin so this combats that